proto/oak_functions/application_config.proto

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright 2024 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.functions.config;
+
+enum HandlerType {
+  // Defaults to WASM.
+  HANDLER_UNSPECIFIED = 0;
+
+  // Use a wasm interpreter to load the module.
+  HANDLER_WASM = 1;
```

Comment:
Maybe we should actually use this to select between Wasmi and Wasmtime at some point too.